tests/test_fx/test_comm_size_compute.py

```diff
@@ -31,12 +35,12 @@ def forward(self, x):
         return x
 
 
-@pytest.mark.skipif(not META_COMPATIBILITY, reason='torch version is lower than 1.12.0')
 def test_comm_size_compute():
-    from colossalai.fx.profiler import MetaTensor
     model = MLP(MODEL_DIM)
-    input_sample = MetaTensor(torch.rand(BATCH_SIZE, MODEL_DIM, device='meta'), fake_device='cpu')
+    input_sample = torch.rand(BATCH_SIZE, MODEL_DIM, device='meta')
     gm = symbolic_trace(model)
+    if is_compatible:
+        input_sample = MetaTensor(input_sample, fake_device=next(gm.parameters()).device)
```

Comment:
If we have torch12, we can do `MetaInfoProp` with `MetaTensor` with high accuracy. Otherwise, we should use the `MetaInfoProp` for torch11 with low accuracy.